Documentation/compatibility/interop-enable-winrt-iagileobject.md

```diff
@@ -0,0 +1,56 @@
+## .NET Interop will now QI for IAgileObject (a WinRT interface)
+
+### Scope
+Edge
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+When using a WinRT event with a .NET delegate, Windows will QI for IAgileObject.  Prior to this change the runtime would fail that QI and the event could not be subscribed.
```

Comment:
```suggestion
When using a WinRT event with a .NET delegate, Windows will QI for IAgileObject starting with the .NET Framework 4.8.  In previous versions of the .NET Framework, the runtime would fail that QI, and the event could not be subscribed.
```